modules/nw-egress-ips-config-object.adoc

```diff
@@ -4,11 +4,14 @@
 
 [id="nw-egress-ips-config-object_{context}"]
 = EgressIPconfig object
-As a feature of egress IP, the `reachabilityTotalTimeoutSeconds` parameter configures the total timeout for checks that are sent by probes to egress IP nodes. The `egressIPConfig` object allows users to set the `reachabilityTotalTimeoutSeconds` `spec`. If the EgressIP node cannot be reached within this timeout, the node is declared down.
 
-You can increase this value if your network is not stable enough to handle the current default value of 1 second.
+As a feature of egress IP, the `reachabilityTotalTimeoutSeconds` parameter configures the EgressIP node reachability check total timeout in seconds. If the EgressIP node cannot be reached within this timeout, the node is declared down.
 
-The following YAML describes changing the `reachabilityTotalTimeoutSeconds` from the default 1 second probes to 5 second probes:
+You can set a value for the `reachabilityTotalTimeoutSeconds` in the configuration file for the `EgressIPconfig` object. Setting a large value might cause the EgressIP implementation to react slowly to node changes. The implementation reacts slowly for EgressIP nodes that have an issue and are unreachable. 
```

Comment:
```suggestion
You can set a value for the `reachabilityTotalTimeoutSeconds` in the configuration file for the `egressIPConfig` object. Setting a large value might cause the EgressIP implementation to react slowly to node changes. The implementation reacts slowly for EgressIP nodes that have an issue and are unreachable. 
```

Reply:
Um, a bit embarrassing but do mind grabbing my typos in this doc while you are here. I introduced them starting at the title. If not, no worries and I'll clean this up in a follow up PR. I think we should get these two, please. I _think_ I introduced this typo stemming from the title.

Reply:
I see it this way here: https://docs.openshift.com/container-platform/4.14/rest_api/operator_apis/network-operator-openshift-io-v1.html#spec-defaultnetwork-ovnkubernetesconfig-egressipconfig:~:text=Description-,egressIPConfig,-holds%20the%20configuration

Reply:
Not at all. Easily happens. My brain did not see these differences either until you pointed them out. Updated.